packages/api/src/external/commonwell/cq-bridge/coverage-enhancement-get-patients.ts

```diff
@@ -42,5 +45,9 @@ export async function getPatientsToEnhanceCoverage(cxIds: string[]): Promise<Pat
       createdAt: { [Op.lte]: earliestDate },
     },
   });
-  return patientsWithIds.map(p => ({ cxId: p.cxId, id: p.id }));
+  return patientsWithIds.map(p => ({
+    cxId: p.cxId,
+    id: p.id,
+    cwLinkStatus: (p.data.externalData?.COMMONWELL as PatientDataCommonwell).status as CWLinkStatus,
```

Comment:
efficient to get status here instead of reiterating through patients and getting it again